_templates/how-to.md

```diff
@@ -1,26 +1,72 @@
+position: [Keep this field blank for now!!! Integer specifying the display position of the article in the navigation. If omitted, the articles are ordered alphabetically based on file name]
 tags: [Relevant keywords. Currently not used but always fill in for future proofness]
-<!-- TO DO: Do we we need the below fields for How-To articles? -->
-teampulseid: [number]
-ticketid: [number]
-pitsid: [number] <!-- Not sure we need this, as pits item are related to teampulse anyway -->
+teampulseid: [A number, representing the TeamPulse ID of the issue\question. Leave this blank if the issue\question doesn't exist as an item in TeamPulse]
+ticketid: [A number, representing the ticket ID associated with this issue\question. The field can contain more than one ticket IDs (e.g. several clients have reported it). Leave this blank if the issue\question is not related to a support case]
+pitsid: [A number, representing the public issue tracker ID associated with the issue\question. Leave this blank for now as PITS is not developed yet]
 
 ---
-
-<!-- Environment Table -->
-<!-- TO DO:
-     We need to check how such can be developed according to our needs:
- 	 We don't need Header roll, but rather a title: Environment. -->
-| Service:     | [AppBuilder], [Backend Services], [AppManager] |
-| Other stuff: | [Other stuff here]                             |
-
-
+                    
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code, Views, Distribution, Data, Users, Notifications, Business Logic, Analytics, Feedback, Samples, Telerik Platform General, 3rd party product</td>	
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>[specify explicitly]</td>	
+  </tr>
+  <tr>
+    <td>Plugin(s)</td>
+    <td>[specify explicitly]</td>	
+  </tr>
+  <tr>
+    <td>Mobile Development Type</td>
+    <td>Hybrid (Cordova app), Native (NativeScript app)</td>	
+  </tr>
+  <tr>
+    <td>Code (AppBuilder) client</td>
+    <td>In-Browser, Extension for Visual Studio, Universal Desctop, Windows, Command-line interface, Sublime Text</td>	
+  </tr>
+  <tr>
+    <td>Mobile OS</td>
+    <td>iOS, Android, Windows Phone 8</td>	
+  </tr>
+  <tr>
+    <td>Mobile OS version</td>
+    <td>[specify explicitly]</td>	
+  </tr>  
+  <tr>
+    <td>Backend Services Area</td>
+    <td>JavaScript SDK (Hybrid app), JavaScript SDK (Web app), JavaScript SDK (Cloud Code), .NET SDK (Web app), Native iOS app, Native Android app, Native Windows Phone app</td>	
+  </tr>
+  <tr>
+    <td>Cordova\NativeScript framework version</td>
+    <td>[specify explicitly]</td>	
+  </tr>
+  <tr>
+    <td>Browser</td>
+    <td>[specify explicitly]</td>	
+  </tr>
+  <tr>
+    <td>Browser version</td>
+    <td>[specify explicitly]</td>	
+  </tr>
+  <tr>
+    <td>.NET framework version</td>
+    <td>[specify explicitly]</td>	
+  </tr>
+  <tr>
+    <td>Visual Studio version</td>
```

Comment:
Version

Honestly, I'm starting to wonder about the capitalization of the header column... The first entries use title style, then we jump to sentence style. Perhaps sentence style would be more readable? In any case, we should be consistent.